paddle/fluid/operators/matrix_power_op.h

```diff
@@ -0,0 +1,277 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#pragma once
+
+#include <memory>
+#include <vector>
+#include "paddle/fluid/framework/op_registry.h"
+#include "paddle/fluid/framework/tensor_util.h"
+#include "paddle/fluid/operators/math/blas.h"
+#include "paddle/fluid/operators/math/matrix_inverse.h"
+#include "paddle/fluid/platform/for_range.h"
+
+namespace paddle {
+namespace operators {
+
+using Tensor = framework::Tensor;
+
+template <typename T>
+struct EyeFunctor {
```

Comment:
Maybe another name?

Reply:
This naming method refers to that of [eye_op.h L27](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/eye_op.h#L27) and [cholesky_op.h L88](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/cholesky_op.h#L88). If it's necessary to rename this functor, what about `IdentityMatrixFunctor`?

Reply:
Done.